core/arch/arm/tee/pta_socket.c

```diff
@@ -0,0 +1,330 @@
+/*
```

Comment:
there is no real need to store "pseudo-TAs" in sta/ sub directories.
As far as this pTA (sic) provides a "TEE" service, it's fine locating it in tee/, and it depends on arch/arm support (thread_rpc), so fine for arch/arm/tee/.
Note all "static TAs" currently in optee_os are all in sta/ subdirs and are all test oriented (+ stats), not effective services.